src/lib/Libifl/pbsD_connect.c

```diff
@@ -645,7 +645,8 @@ pbs_connect_noblk(char *server, int tout)
 	struct batch_reply *reply;
 	char server_name[PBS_MAXSERVERNAME+1];
 	unsigned int server_port;
-	struct addrinfo *aip, *pai = NULL;
+	struct addrinfo *aip;
+	struct addrinfo *pai = NULL;
```

Comment:
Is this intentional?

Reply:
You told me that if i was going to set one of the variables, to put them on separate lines c:

Reply:
Oh, sorry, I missed that both variables were there before.  I thought you were adding one.